lib/bom.gradle

```diff
@@ -1,32 +1,36 @@
 configure(projectsWithFlags('bom')) {
     apply plugin: 'java-platform'
     apply plugin: 'maven-publish'
+}
 
-    publishing {
-        publications {
-            bom(MavenPublication) {
-                from components.javaPlatform
+afterProjectsWithFlag('java') { Set<Project> projects ->
```

Comment:
Note for other reviewers: It is good to review with hiding white space changes.

Reply:
That's a brilliant tip! 😄